0/GW/Next_Event_Weather.ts

```diff
@@ -127,13 +129,13 @@ async function searchLocationKey(
   location: string,
 ): Promise<any[]> {
   try {
-    return ((await got
+    return await got
       .get('http://dataservice.accuweather.com/locations/v1/search', {
         cache: httpRequestCache,
         retry: 0,
         searchParams: { apikey: apiKey, q: location },
       })
-      .json<any[]>()) as unknown) as any[];
```

Comment:
Without this, ioBroker shows this error during compilation:

```
script.js.GW.Next_Event_Weather: TypeScript compilation failed: return await got ^ ERROR: Type 'CancelableRequest<unknown>' is missing the following properties from type 'any[]': length, pop, push, concat, and 28 more.
```

Reply:
These are the compiler options used as it seems:

https://github.com/ioBroker/ioBroker.javascript/blob/00c965e9ec10d13bff758472b8f8c2792abcca0d/main.js#L104

Reply:
Thx for the link.
Helps to configure tsc according to ioBroker